spec.json

```diff
@@ -1,7 +1,7 @@
 {
   "nio/PickleBlock": {
```

Comment:
And I'm thinking we should remove the word `Block` from the end of these block names. None of the other ones have that.